.agents/skills/consolidation/nemoclaw-sprint-review/collect_sprint_items.py

```
@@ -0,0 +1,136 @@
+                  }}
+                }}
+                pageInfo {{
+                  hasNextPage
+                  endCursor
+                }}
+              }}
+            }}
+          }}
+        }}'''
+
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)